README.md

```diff
@@ -1,6 +1,171 @@
-# Wagtail-Template
+<p align="center">
+  <a href="https://www.aichner-christian.com/" target="_blank" rel="noopener noreferrer">
+    <img src="https://www.aichner-christian.com/img/logo/logo_web.png" alt="Agency Logo" height="150">
+  </a>
+</p>
 
-This repository serves as an entry point for our future Wagtail projects.
+<h3 align="center">Official Wagtail Template</h3>
+
+<p align="center">
+  This is the official template repository for Wagtail projects of the Advertisement Agency Christian Aichner.
+  <br>
+  <br>
+  <a href="https://github.com/aichner/Wagtail-Template/issues/new?template=bug_report.md">Report bug</a>
+  ·
+  <a href="https://github.com/aichner/Wagtail-Template/issues/new?template=feature_request.md">Request feature</a>
+  ·
+  <a href="https://www.aichner-christian.com/damn/dev">Blog</a>
+</p>
+
+## Table of contents
+
+- [Quick start](#quick-start)
+- [Status](#status)
```

Comment:
Please remove the reference to the removed section.

Reply:
yikes xD

Reply:
![image](https://user-images.githubusercontent.com/21159914/63637497-e8790380-c67c-11e9-8c0e-9ef0ce27ea30.png)